connect/src/protocols/tokenBridge/tokenTransfer.ts

```diff
@@ -343,13 +343,11 @@ export namespace TokenTransfer {
     receipt: TokenTransfer.TransferReceipt<SC, DC>,
     timeout: number = DEFAULT_TASK_TIMEOUT,
     fromChain?: ChainContext<N, SC>,
-    toChain?: ChainContext<N, DC>,
```

Comment:
why take this out? if anyone is providing it, they'll be sad

Reply:
i took it out because we don't require it. we should always be able to get it from the attestation. so why let people pass it in, what does that do for them? there's a chance they pass in the wrong chain and it gets fucky

Reply:
just looked at this again and made that change actually :+1:
thanks

Reply:
It provides the ability to re-use the same chain in the case that its already created with a specific RPC